be/src/util/ratelimit.h

```diff
@@ -0,0 +1,17 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+#pragma once
+
+#include <atomic>
+
+#include "util/time.h"
+
+#define RATE_LIMIT(func, interval_ms)          \
+    {                                          \
+        static std::atomic<int64_t> last;      \
```

Comment:
Can it check the tablet? Make it print a log for different tablets at intervals.

Reply:
Done, Add macro RATE_LIMIT_BY_TAG.